packager/react-packager/src/Resolver/polyfills/babelHelpers.js

```diff
@@ -19,6 +19,12 @@
 
 var babelHelpers = global.babelHelpers = {};
 
+babelHelpers.typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
+  return typeof obj;
+} : function (obj) {
+  return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj;
```

Comment:
Can you add an `obj !== Symbol.prototype` comparison too? See https://github.com/babel/babel/blob/master/packages/babel-helpers/src/helpers.js#L8